### PR TITLE
SpMV fix

### PR DESCRIPTION
**Pull request by henryre**
There was an issue with `SparseLogisticRegression` when the data matrix had an all-zero row

### CODE REVIEW

**Comment by henryre** [comment]
LGTM!

File: snorkel/learning/logistic_regression.py [hunk 1]
```diff
@@ -179,13 +179,22 @@ def _check_input(self, X):
         return X.tocsr()
 
     def _batch_sparse_data(self, X):
-        """Convert sparse batch matrix to sparse inputs for embedding lookup"""
+        """Convert sparse batch matrix to sparse inputs for embedding lookup
```
Comment (ajratner) [inline snorkel/learning/logistic_regression.py]: What the hell is this

Comment (ajratner) [inline snorkel/learning/logistic_regression.py]: Please use proper indentation

File: snorkel/learning/logistic_regression.py [hunk 2]
```diff
@@ -179,13 +179,22 @@ def _check_input(self, X):
...
+            Notes: https://github.com/tensorflow/tensorflow/issues/342
+        """
         if not issparse(X):
             raise Exception("Matrix X must be scipy.sparse type")
         X_lil = X.tolil()
         indices, ids, weights = [], [], []
         max_len = 0
         for i, (row, data) in enumerate(zip(X_lil.rows, X_lil.data)):
+            # Dummy weight for all-zero row
```
Comment (ajratner) [inline snorkel/learning/logistic_regression.py]: Unkind, nasty words.  Sad.

Comment (henryre) [inline snorkel/learning/logistic_regression.py]: You're bad at deals.

**Pull request merged**